src/Libraries/Nop.Core/Domain/Security/EncryptionFormat.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Nop.Core.Domain.Security {
+    public enum EncryptionFormat {
+        TripleDes = 0,
```

Comment:
I'm not a maintainer here, but why add support for TripleDes?  Isn't it much less secure than AES?